cogs/spielereien.py

```diff
@@ -0,0 +1,163 @@
+        """
+        Replies the needed time to gather and answer the message.
+        """
+        reply: Message = await ctx.reply("Pong")
+        msg: Message = ctx.message
+        await reply.edit(content=f"{reply.created_at.timestamp() - msg.created_at.timestamp()}")
+
+    @command()
+    async def man(self, ctx: Context, manpage: str):
+        """
+        Fetches a requested manpage.
+
+        Args:
+            ctx: The command context provided by the discord.py wrapper.
+
+            manpage: The command to search for.
+
+        Replies:
+            The Manpage.
+
+        Raises:
+            ManPageNotFound
+        """
+        msg: Message = ctx.message
+        mentions: set = set(msg.mentions)
+        cmd_sec = manpage.split("#")
+        cmd = cmd_sec[0]
+        if len(cmd_sec) > 1:
+            subsection = f"#{cmd_sec[1]}"
+        else:
+            subsection = ""
+
+        reference: MessageReference = msg.reference
+        if reference:
+            channel: TextChannel = ctx.channel
+            replied_to: Message = await channel.fetch_message(reference.message_id)
+            mentions.add(replied_to.author)
+
+        ping = ""
+
+        for mention in mentions:
+            if isinstance(mention, Member):
+                mention: Union[Member, User]
+                ping += f"<@{mention.id}> "
+
+        result: list = list(await asyncio.gather(
+            self.get_page(f"https://man.openbsd.org/{cmd}", 1),
+            self.get_page(f"https://wiki.archlinux.org/index.php/{cmd}", 2)
+        ))
+
+        result.sort(key=lambda x: x[0])
+
+        for _, page in result:
+            if page.ok:
+
+                if mentions:
+                    embed: Embed = Embed(title="Manpage",
+                                         # TODO: This message should be stored into the database
+                                         description=f"I was asked to show you this man page:\n"
+                                                     f"It contains useful information that will hopefully help you.\n"
+                                                     f"On your UNIX/UNIX-like system you can probably run "
+                                                     f"`$ man {cmd}` to receive this manpage.\n"
+                                                     f"For now you can also click on this link to gather the "
+                                                     f"information.\n"
+                                                     f"{page.url}{subsection}")
+                    await ctx.reply(embed=embed, content=ping)
+                else:
+                    embed: Embed = Embed(title="Manpage",
+                                         description=f"On your UNIX/UNIX-like system you can probably run:\n"
+                                                     f"`$ man {cmd}`\n"
+                                                     f"You can also use this site: {page.url}{subsection}")
```

Comment:
extract these 3 String of the description lines since they are (or at least should be) the same no matter if there is a mention or not